### PR TITLE
Add index.html and style.css file to the desktop version

### PR DESCRIPTION
**Pull request by Etsubdink-Demeke**
This pull request is created to the desktop version of my portifolio for adding the index.html,the style.css and all the other resourses.

### CODE REVIEW

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
Hi @Etsubdink-Demeke

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# ⚠️ Invalid Code Review Request ⚠️ 

You have submitted a project, that does not have a descriptive Pull Request (PR). The PR descriptions should contain a summary of the changes you have in the project/PR. This gives the reviewer an idea of the changes that took place in this particular PR. Kindly fix this before requesting another review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @stanleySimeon in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+            <li> <a href=""><img src="images/icons/Vector (2).png" alt="Twitter icon"></a></li>
+            <li><a href=""><img src="images/icons/Vector (3).png" alt="Twitter icon"></a></li>
+          </ul>
+        </div>
+      </div>
+      </section>
+      </section>
+      <section id="work">
+         <div class="cards">
+            <div class="screenshoot"><img src="images/Portoflio Card/Snapshoot Portfolio.jpg"></div>
```
Comment (wickathou) [inline index.html:44]: - [ ] **Required:** Your work section background has a different color and the presentation of the cards in your design has difference on their proportions to the figma design

Your design
image

Figma design
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+            <div class="description">
+              <h2>Tonic</h2>
+              <ul class="cards__elements--items">
+                <li id="cards__elements--item">Canopy</li>
+                <li>backend</li>
+                <li>2015</li>
```
Comment (wickathou) [inline index.html:50]: - [ ] **Required:** These elements have a different color on the figma design, please update it to match the figma file

Your design
image


Figma design
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+          <ul>
+            <li><a href=""><img src="images/icons/Vector.png" alt="Twitter icon"></a></li>
+            <li><a href=""><img src="images/icons/Linkedin icon.png" alt="Twitter icon"></a></li>
+            <li><a href=""><img src="images/icons/Vector (1).png" alt="Twitter icon"></a></li>
+            <li> <a href=""><img src="images/icons/Vector (2).png" alt="Twitter icon"></a></li>
+            <li><a href=""><img src="images/icons/Vector (3).png" alt="Twitter icon"></a></li>
+          </ul>
+        </div>
+        
+        <button>Get my resume</button>
```
Comment (wickathou) [inline index.html:141]: - [ ] **Required:** Your buttons across the site don't look like the figma design, update it to match the provided design

- [ ] **Required** The 'LET's CONNECT' link has a different font and color to the one provided, please update it to match the figma design

Your design
image

Figma design
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+          </li>
+          <li class="button">
+            <button type="submit">Get in touch</button>
+          </li>
+            
+        </form>
+
+      </div></div>
+    </section> 
+    </section>
```
Comment (wickathou) [inline index.html:227]: - [ ] **Required:** Your 'contact me' section has a different alignment to the one shown on the figma design and the blue background has a larger height than the figma design

Your design
image

Figma design
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+     <section class="container">
+      <section  id="portifolio">
+      <header class="nav">
+        <div id="nav__logo"><a href="#nav__logo">My Logo</a></div>
+        <nav class="nav__elements">
+          <div class="nav__elements--element"><a href="#work">Portifolio</a></div>
+          <div class="nav__elements--element"><a href="#about me">About</a></div>
+          <div class="nav__elements--element"><a href="#contact-page">Contact</a></div>
+        </nav>
+      </header>
```
Comment (wickathou) [inline index.html:23]: [Optional] The sticky navbar is not a requirement but it's great to see you experimenting with it. One thing you could improve is to ensure that it is properly centered, currently in has more empty space on the right side

image

File: package.json
Comment (wickathou) [inline package.json:1]: [Optional] For your next project, consider using `npm init -y` to initialize your package.json properly

**Review by wickathou — CHANGES_REQUESTED** [review]
Hi @Etsubdink-Demeke  , 

>You are almost there!
Great effort so far, but unfortunately there are some issues that you need to review in order to get your project approved, but you got this!

test
>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
[Optional] Your CSS file has multiple selectors that repeatedly use the `font-family: 'Poppins', sans-serif;` property to assign the font to individual parts of your code. To keep your code DRY, consider using a global selector `*` as your template uses 'Poppins on all parts of the design

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
<br/>
### Consider also:
- Exploring why initiating your project with `npm init` is a good practice
<br/>

## Highlights
- Interesting use of the `position: fixed` property on your navbar 🏅
- Using a license in the project 💯
- Descriptive commit messages ✨
<br/>
Feel free to leave any questions or comments in the PR thread mentioning me so I can see the notification and reply `@wickathou`.

You can also try reaching me on Slack, looking for me as `@Javier Hernandez`

Cheers and Happy coding!👏👏👏
<br/>
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by NatiG25 — COMMENTED** [review]
## :red_circle: INVALID PULL REQUEST :red_circle: 

## Hi @Etsubdink-Demeke :wave:, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 
:x: Didn't pass all linter checks
:x: [OPTIONAL] Kindly, create a LICENSE file then add the content present at the README file (License section)  & create a link (MIT) that'll refer to the LICENSE file. This way your README file will match Microverse's README template. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+              
+              
+          
+        
+        </ul>    
+        </div>
+      </div>
+    </div>
+    </section>
+    <section id="contact-page">
```
Comment (Efiamotu-1) [inline index.html:203]: - [ ] You are yet to implement the requirement made by the previous reviewer provided in this [link](https://github.com/Etsubdink-Demeke/portifolio-desktop-version/pull/1#discussion_r1186466593). The contact me section of your portfolio is not properly centered in relation to the button as well the background image is also not properly placed to reflect closely the given figma design
- [ ] It is important to create your portfolio project in the best possible way to closely match the provided design as this shows your attention to details as a frontend web developer so please kindly work on properly contering the contents of your contact me on desktop version as well as plaving the background images correctly. :+1:

File: index.html [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              <ul class="cards__elements--list">
+                <li>html</li>
+                <li>css</li>
+                <li>JavaScript</li>
+              </ul>
+              <button>see project</button>
+            </div>
+        </div>
+        <div class="cards">
```
Comment (Efiamotu-1) [inline index.html:66]: - [ ] The previous reviewers requirement in this [link](https://github.com/Etsubdink-Demeke/portifolio-desktop-version/pull/1#discussion_r1186459734) has also not been implemented. below is the current design of one of your cards on desktop view. please kindly make sure to align the contents of each card properly using the respective css properties as it is always important to keep your project professional and attractive to users and recruiters. :+1:  
- your design
car-s

File: index.html
Comment (Efiamotu-1) [inline index.html:1]: - your design on the mobile version is still very inconsistent with the design on figma and as pointed by your previous reviewer in this [link](https://github.com/Etsubdink-Demeke/portifolio-desktop-version/pull/1#discussion_r1186454546). please kindly readjust the styling of your portfolio page on a mobile device to match industry standards of mobile responsive web pages. :+1:

File: index.html [hunk 8]
```diff
@@ -0,0 +1,234 @@
...
+            <div class="description">
+              <h2>Multi-Post-Stories</h2>
+              <ul class="cards__elements--items">
+                <li id="cards__elements--item">FACEBOOK</li>
+                <li>Full Stack Dev</li>
+                <li>2015</li>
+              </ul>
+              <p>Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
+              <ul class="cards__elements--list">
```
Comment (Efiamotu-1) [inline index.html:75]: - [ ] please make sure to use the correct font color for the technologies as shown in the figma design below, you can find the correct fonts from the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=34-848&t=iCVc0ujmBGGSZMXT-0) 
- figma design
con-f
- your design
con-s

File: styles.css
Comment (Efiamotu-1) [inline styles.css:1]: - your css styling on line `54` as shown below which gives the nav 100% width is after the overall look of your portfolio on both the mobile and desktop version causing it to overflow the viewport please look into this and make the proper use of the css property to style the desktop version of your project. :+1: 
```
  .nav {
    width: 100%;
  }
```

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @Etsubdink-Demeke , 



Great work on making the changes requested by a previous reviewer 👏 Your Code is Awesome! 💪
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Especially if that's is your first project 1️⃣. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better job** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (wayungi) [reply index.html:232]: - [ ] This has been highlighted before. I will try and use a diagram to explain the point. Please center the form below the text. 😄 
form

File: index.html (hunk 7 above)
Comment (wayungi) [reply index.html:66]: - [ ] Good work in trying to align the contents properly. You have not implemented the space between the image and the content as in Figma. Please look at the illustration below and add the required space. **Note** This applies to all your cards
card

File: index.html
Comment (wayungi) [reply index.html:1]: - [ ] As suggested by the previous reviewer, your page does not fit on the mobile view. Please let work on this using media queries to apply different rule sets based on device width. 😅

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @Etsubdink-Demeke , 

image

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
- [ ] All linter checks are passing ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by shella12 — COMMENTED** [review]
Hi <STUDENT_MENTION>, 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, 

- [ ] that has linter errors. Please fix the linter errors before submitting for another request

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**